eth/downloader/downloader_test.go

```diff
@@ -752,9 +723,6 @@ func testMultiSynchronisation(t *testing.T, protocol uint, mode SyncMode) {
 func TestMultiProtoSynchronisation68Full(t *testing.T)  { testMultiProtoSync(t, eth.ETH68, FullSync) }
 func TestMultiProtoSynchronisation68Snap(t *testing.T)  { testMultiProtoSync(t, eth.ETH68, SnapSync) }
 func TestMultiProtoSynchronisation68Light(t *testing.T) { testMultiProtoSync(t, eth.ETH68, LightSync) }
-func TestMultiProtoSynchronisation67Full(t *testing.T)  { testMultiProtoSync(t, eth.ETH67, FullSync) }
```

Comment:
Do we want to keep ETH67 and its tests?

Reply:
What is the latest version implemented by besu and Erigon?

Reply:
This is the PR removing it https://github.com/ethereum/go-ethereum/pull/28956

Reply:
PR that reverts the ETH67 removal https://github.com/etclabscore/core-geth/pull/629

Reply:
Ohh, @meowsbits @diega we have to merge this one I think. If yes, then we have to rerun our sync test.